usagov/usagov-2021
# USAGOV-1564 self reflexive link

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1564

## Description

Reinstitute the rel alternate hreflang tags on each page that referred to the current page 
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Goto http://localhost/ and view page source. Make sure the information for the rel link shows as the screenshot below.
- 
<img width="563" alt="Screenshot 2024-06-12 at 9 42 53 AM" src="https://github.com/usagov/usagov-2021/assets/5551880/e3e47f26-c61d-4a75-a6d8-ae972bc4cebd">

- Goto http://localhost/elected-officials and view page source. Make sure the information for the rel link shows as the screenshot below.
- 
<img width="637" alt="Screenshot 2024-06-12 at 9 42 33 AM" src="https://github.com/usagov/usagov-2021/assets/5551880/3a83e463-5dcc-4e29-bf76-4b249cbe3a3f">



## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
